hyper-boring/Cargo.toml

```diff
@@ -21,7 +21,7 @@ http = "0.2"
 hyper = { version = "0.14", default-features = false, features = ["client"] }
 linked_hash_set = "0.1"
 once_cell = "1.0"
-boring = { version = "1.1.0", path = "../boring" }
+boring = { version = "2", path = "../boring" }
```

Comment:
I think we need a major bump for those too: considering that you can't use 2 versions of boring side by side, this implies that in order to use this version of `hyper-boring` you would need to bump boring dep for your app, otherwise it won't compile. So, it's technically a breaking change too.

Reply:
Oh - we can fix that by allowing either the old or new version of `boring` I think. Let me look up the semver constraint for that.

Reply:
Ok, done.